lib/dynflow/world.rb

```diff
@@ -32,7 +33,7 @@ def default_options
     end
 
     def clock
-      @clock ||= Clock.new(logger)
+      @clock_barrier.synchronize { @clock ||= Clock.new(logger) }
```

Comment:
Could we set up the clock in the `initialize` method? Then we can avoid the barrier.